### PR TITLE
Merge schema into gouvfr

### PR DESCRIPTION
**Pull request by quaxsze**

### CODE REVIEW

File: theme/js/components/dataset/schema-modal.vue [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<template>
+  <vue-final-modal class="modal-wrapper">
+    <div class="modal-body" style="color: #000000">
```
Comment (abulte) [inline theme/js/components/dataset/schema-modal.vue:3]:
```suggestion
    <!-- FIXME: style in css -->
    <div class="modal-body" style="color: #000000">
```

**Review by abulte — COMMENTED** [review]
Add a setting to bypass this if set to False

**Review by maudetes — APPROVED** [review]

File: udata_gouvfr/frontend/__init__.py [hunk 2]
```diff
@@ -101,12 +101,14 @@ def init_app(app):
 
     from . import helpers, error_handlers  # noqa
 
+    if app.config['RESOURCES_SCHEMAS_ENABLED']:
```
Comment (abulte) [inline udata_gouvfr/frontend/__init__.py:104]:
```suggestion
    if app.config['RESOURCES_SCHEMAGOUVFR_ENABLED']:
```

File: udata_gouvfr/settings.py [hunk 3]
```diff
@@ -2,6 +2,7 @@
 Default settings for udata-gouvfr
 """
 
+RESOURCES_SCHEMAS_ENABLED = True
```
Comment (abulte) [inline udata_gouvfr/settings.py:5]:
```suggestion
# TODO: move this back to an extension
RESOURCES_SCHEMAGOUVFR_ENABLED = True
```

**Review by abulte — APPROVED** [review]

**Pull request merged**